src/server/Elsa.Server.Api/Models/WorkflowTestMessage.cs

```diff
@@ -0,0 +1,14 @@
+namespace Elsa.Server.Api.Models
+{
+    public class WorkflowTestMessage
+    {
+        public string? Path { get; set; }
```

Comment:
What's this used for?

Reply:
This one is used to generate the full path that can be copied/pasted to a browser window or Postman to execute suspended HttpEndpoint.

Reply:
OK.

The issue with that is that `Path` is an attribute that specifically belongs to the HTTP Endpoint activity, while `WorkflowTestMessage` is activity type-agnostic (it shouldn't care about specific types of activities).

In other words, this is tight-coupling HTTP endpoint aspects with this workflow test model.

Imagine if Elsa didn't ship with the HTTP Endpoint activity but instead provided it as a separate package. How would you solve this problem?

Similarly, there might already exist 3rd party activities that offer their own version of HTTP Endpoints. Elsa should offer them a way to somehow display the test URL.

If you look at it from that perspective, you'll see it's necessary to handle this in a more extensible way.